examples/github-complete/main.tf

```diff
@@ -69,10 +69,11 @@ module "atlantis" {
   atlantis_allowed_repo_names = var.allowed_repo_names
 
   # ALB access
-  alb_ingress_cidr_blocks = [var.personal_ip]
-  alb_logging_enabled     = true
-  alb_log_bucket_name     = module.atlantis_access_log_bucket.this_s3_bucket_id
-  alb_log_location_prefix = "atlantis-alb"
+  alb_ingress_cidr_blocks         = [var.personal_ip]
```

Comment:
`var.personal_ip` is a bit _small_. It can be the whole set of CIDR blocks. Let's rename it to match, something like - `var.alb_ingress_cidr_blocks`

Reply:
updated